pytorch_lightning/loggers/neptune.py

```diff
@@ -296,113 +565,46 @@ def version(self) -> str:
         Returns:
             The id of the experiment if not in offline mode else "offline-id-1234".
         """
-        if self.offline_mode:
-            return "offline-id-1234"
-        return self.experiment.id
+        return self.run._short_id  # skipcq: PYL-W0212
```

Comment:
```suggestion
        return self.run._short_id
```
what's this?

Reply:
Ok, I see that we have problem here...

`Neptune` has project with attached experiments.
Let's assume that we have and project with name `Domestic Market` with abbreviation `DOM`.

Then experiments created for this project will have auto generated short ids like follows: `DOM-1`, `DOM-2`, `DOM-3`...

So I guess that `logger.name` should return those values accordingly instead of "NeptuneLogger".
And `logger.version` should return the same values or integers like: 1, 2, 3... Which one do you prefer?

Reply:
no I meant the skipcq comment there :)

Reply:
Oh, it's here because your sytylecheck complained on using "private" attribute of `run`.

Unfortunately we have to do it his way, because `_short_id` is always available and doesn't require additional API CALL to obtain id.